### PR TITLE
service: handle BatchCommandsEmptyRequest

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

## What have you changed? (mandatory)
Handle `BatchCommandsRequst::Empty`. This message is generally for tests, make us can focus on transport layer instead of backend logics.

## Relative works
- I suggeste to cherry pick it to release-3.0, so that we can do more tests for release-3.0.

### CODE REVIEW

File: src/server/service/kv.rs [hunk 1]
```diff
@@ -1263,9 +1263,29 @@ fn handle_batch_commands_request<E: Engine>(
...
 }
 
+fn future_handle_empty(
+    req: BatchCommandsEmptyRequest,
+) -> impl Future<Item = BatchCommandsEmptyResponse, Error = Error> {
+    tikv_util::timer::GLOBAL_TIMER_HANDLE
+        .delay(std::time::Instant::now() + std::time::Duration::from_millis(req.get_delay_time()))
+        .map(move |_| {
+            let mut res = BatchCommandsEmptyResponse::new();
+            res.set_test_id(req.get_test_id());
```
Comment (Connor1996) [inline src/server/service/kv.rs]: not set `replay_time` ?

Comment (hicqu) [reply src/server/service/kv.rs]: There is no `replay_time` in response now.

File: src/server/service/kv.rs [hunk 2]
```diff
@@ -1263,9 +1263,29 @@ fn handle_batch_commands_request<E: Engine>(
                 .map_err(|_| GRPC_MSG_FAIL_COUNTER.kv_pessimistic_rollback.inc());
             response_batch_commands_request(id, resp, tx, timer);
         }
+        Some(BatchCommandsRequest_Request_oneof_cmd::Empty(req)) => {
+            let timer = GRPC_MSG_HISTOGRAM_VEC.invalid.start_coarse_timer();
+            let resp = future_handle_empty(req)
+                .map(oneof!(BatchCommandsResponse_Response_oneof_cmd::Empty))
+                .map_err(|_| GRPC_MSG_FAIL_COUNTER.invalid.inc());
+            response_batch_commands_request(id, resp, tx, timer);
```
Comment (overvenus) [inline src/server/service/kv.rs]: Could you add a test?

Comment (hicqu) [reply src/server/service/kv.rs]: There is already a test case in `tests/integrations/server/kv_service.rs`.

Comment (overvenus) [reply src/server/service/kv.rs]: I mean the `Empty` request.

Comment (hicqu) [reply src/server/service/kv.rs]: I think it's not necessary. `Empty` itself is simple enough.

**Pull request merged**